tests/testthat/test-timeseriessummary.R

```diff
@@ -2921,7 +2921,9 @@ test_that('formatAdvReportOptions function returns expected info to display on T
                                "county": "Aroostook County",
                                "description": "Shows relevant information about a timeseries",
                                "title": "Time Series Summary",
-                               "excludeCorrections": "DeleteRegion,",
+                               "requestParameters": {
+                                  "excludedCorrections": [ "DeleteRegion", "AdjustableTrim", "Deviation" ]
```

Comment:
The model appears to have "excludeCorrections" in the json. https://github.com/USGS-CIDA/aqcu-uv-hydro-report/blob/761f4bb64350c8372f46daf88b96f5ad9309fec9/src/main/java/gov/usgs/aqcu/model/UvHydroReportMetadata.java#L19